content/06kapitel.tex

```diff
@@ -6,5 +6,15 @@ \section{Erreichte Ziele}
 
 \section{Analyse der Organisatorischen Änderungen}
 \todo{Änderungen des Ablaufes erläutern und analysieren}
+
+\subsection{Leitlinie für gemeinsame Arbeit am Quellcode}
+Um die Zusammenarbeit zu vereinheitlichen wurde dafür zu Beginn des Projekts eine Leitfaden entwickelt.
+Dieser umfasst wie der Code dokumentiert werden soll, Regeln für die Arbeit mit dem Versionskontrollsystem Git 
+und wie Code anderer überprüft ("reviewed") werden soll. Mit Hilfe der Leitlinie wurde eine Checkliste entwickelt 
+welche von jedem Entwickler ausgefüllt werden muss, wenn er seinen Code zur Überprüfung freigibt. Richtig
+eingesetzt konnte eine aktive Umsetzung der Leitlinie die Entwicklungsgeschwindigkeit vorantreiben. Die 
+Übereinstimmung auf eine solche Richtlinie hatte dennoch auch Nachteile. Teilweise dauerte die Überprüfung von 
+anderem Code zu lang, was wiederum abhängige Tickets und somit den Entwicklungsprozess blockierte.
```

Comment:
könntest du evtl. noch die guidelines in den Anhang einfügen?

Reply:
als text oder als PDF? :)